manual_jb/content/notebooks_other.md

```diff
@@ -34,6 +34,7 @@ Note that some Notebooks might not work with the latest version of pyIncore.
 [tornadoepn_dmg.ipynb](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/tornadoepn_dmg.ipynb) <br />
 [transportation_recovery.ipynb](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/transportation_recovery.ipynb) <br />
 [water_facility_dmg.ipynb](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/water_facility_dmg.ipynb) <br />
+[wfn_functionality.ipynb](https://github.com/IN-CORE/incore-docs/blob/master/notebooks/wfn_functionality.ipynb) <br />
```

Comment:
this should be main not master. The link will work once we release.

Reply:
I have an issue to fix the others that should also be main and not master. I will do that right before we release.